MineInAbyss-API/src/main/java/com/derongan/minecraft/mineinabyss/API/Relic/Behaviour/OnDamageRelicBehaviour.java

```diff
@@ -0,0 +1,7 @@
+package com.derongan.minecraft.mineinabyss.API.Relic.Behaviour;
+
+import org.bukkit.event.entity.EntityDamageEvent;
+
+public interface OnDamageRelicBehaviour extends RelicBehaviour{
```

Comment:
Why is it called `OnDamageRelicBehaviour` if it has a `EntityDamageEvent`? `OnDamageRelicBehaviour` implies the relic should be taking damage (i.e. losing durability), is this what this class is for or is it just to see when an entity has been damaged? If it's the latter, should we rename it to something like `OnRelicDamageEntityBehaviour`.

Reply:
The name of the behaviour reflects what the behaviour does, not what the event does. Multiple behaviours can correspond to one event.